spec/classes/sensu_init_spec.rb

```diff
@@ -402,7 +402,7 @@
           context "when #{var_name} (#{type}) is set to invalid #{invalid} (as #{invalid.class})" do
             let(:params) { [mandatory_params, var[:params], { :"#{var_name}" => invalid, }].reduce(:merge) }
             it 'should fail' do
-              expect { should contain_class(subject) }.to raise_error(Puppet::Error, /#{var[:message]}/)
+              expect { should contain_class(subject) }.to raise_error(Puppet::PreformattedError)
```

Comment:
@Phil-Friderici I couldn't do any better that this.
To match the actual error messages according to data type would need a refactoring of the test code.... maybe later ;-)